doc/SimpleWeatherService.md

```diff
@@ -28,7 +28,7 @@ The byte array must contain the following data:
 
  - [0] : Message type = `0`
  - [1] : Message version = `0`
- - [2][3][4][5][6][7][8][9] : Timestamp (64 bits UNIX timestamp, number of nanoseconds elapsed since 1 JAN 1970)
+ - [2][3][4][5][6][7][8][9] : Timestamp (64 bits UNIX timestamp, number of seconds elapsed since 1 JAN 1970)  in local time (the same timezone than the one used to set the time)
```

Comment:
```suggestion
 - [2][3][4][5][6][7][8][9] : Timestamp (64 bits UNIX timestamp, number of seconds elapsed since 1 JAN 1970)  in local time (the same timezone as the one used to set the time)
```

Reply:
Done